CMakeLists.txt

```diff
@@ -64,18 +67,45 @@ endif()
 # Installation
 #
 
-include(GNUInstallDirs)
```

Comment:
Why did you remove this?

Reply:
Moved to line 9 because it's already needed in line 22 :)